pkg/daemon/ceph/client/config_test.go

```diff
@@ -49,19 +48,14 @@ func TestCreateDefaultCephConfig(t *testing.T) {
 	}
 
 	// start with INFO level logging
-	context := &clusterd.Context{
-		LogLevel: capnslog.INFO,
-	}
+	context := &clusterd.Context{}
 
 	cephConfig, err := CreateDefaultCephConfig(context, clusterInfo)
 	if err != nil {
 		t.Fatalf("failed to create default ceph config. %+v", err)
 	}
 	verifyConfig(t, cephConfig, clusterInfo, 0)
 
-	// now use DEBUG level logging
-	context.LogLevel = capnslog.DEBUG
-
```

Comment:
This file was the only usage of `LogLevel` from `clusterd.Context`, so I removed it.